recipes/recipes_emscripten/symengine/recipe.yaml

```diff
@@ -1,37 +1,38 @@
 context:
   name: symengine
-  version: 0.12.0
+  version: 0.13.0
 
 package:
   name: ${{ name|lower }}
   version: ${{ version }}
 
 source:
-  sha256: 1b5c3b0bc6a9f187635f93585649f24a18e9c7f2167cebcd885edeaaf211d956
-  url: https://github.com/symengine/symengine/releases/download/v${{ version }}/symengine-${{ version }}.tar.gz
+  sha256: f46bcf037529cd1a422369327bf360ad4c7d2b02d0f607a62a5b09c74a55bb59
+  url: https://github.com/symengine/symengine/releases/download/v${{ version }}/symengine-${{
+    version }}.tar.gz
 
 build:
   number: 0
   script: |
-      emcmake cmake \
-        -DBUILD_TESTS=OFF \
-        -DBUILD_BENCHMARKS=OFF \
-        -DINTEGER_CLASS=boostmp \
-        -DCMAKE_PREFIX_PATH=$PREFIX \
-        -DCMAKE_INSTALL_PREFIX=$PREFIX \
-        -DCMAKE_INSTALL_LIBDIR=lib \
-        -DBUILD_FOR_DISTRIBUTION=yes \
-        -DBUILD_SHARED_LIBS=no \
-        -DBoost_INCLUDE_DIR=$BUILD_PREFIX/include \
-        .
-      emmake make install
+    emcmake cmake \
+      -DBUILD_TESTS=OFF \
+      -DBUILD_BENCHMARKS=OFF \
+      -DINTEGER_CLASS=boostmp \
+      -DCMAKE_PREFIX_PATH=$PREFIX \
+      -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DBUILD_FOR_DISTRIBUTION=yes \
+      -DBUILD_SHARED_LIBS=no \
+      -DBoost_INCLUDE_DIR=$BUILD_PREFIX/include \
+      .
+    emmake make install
 
 requirements:
   build:
-    - ${{ compiler("cxx") }}
-    - cmake
-    - make
-    - boost-cpp
+  - ${{ compiler("cxx") }}
+  - cmake
+  - make
+  - boost-cpp
 
```

Comment:
Well as of now you can just check if we have the neccessary library in place. Add this here should suffice
```
  tests:
  - package_contents:
      files:
        - lib/libsymengine .a     
```